ups/eupspkg.cfg.sh

```diff
@@ -67,4 +67,9 @@ install()
 	default_install
 
 	cp -r include "$PREFIX/"
+	if [[ $OSTYPE == darwin* ]]; then
+		cd lib
+		rm libgalsim.dylib
+		ln -s libgalsim.*.dylib libgalsim.dylib
```

Comment:
Maybe add `cd ..` after the `ln` so that we don't end up in a different directory if people add things later on.

Reply:
Good point.